.github/workflows/build.yml

```
@@ -0,0 +1,59 @@
+    paths:
+      - '.github/workflows/build.yml'
+      - 'module/**'
+      - 'update.json'
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)